lib/conditional.module.ts

```diff
@@ -0,0 +1,24 @@
+import { ModuleMetadata } from '@nestjs/common';
+import { ConfigModule } from '.';
```

Comment:
```suggestion
import { ConfigModule } from './config.module.ts';
```

Reply:
Because my text editor auto-imported and I didn't double check it :sweat_smile: